cmd/agent/app/run.go

```diff
@@ -162,6 +168,18 @@ func StartAgent() error {
 			config.Datadog.GetBool("log_to_console"),
 			config.Datadog.GetBool("log_format_json"),
 		)
+
+		//Setup JMX logger
+		err = config.SetupLogger(
```

Comment:
Here you overwrite `err` without having checked wether it contains an error or not.